code/src/components/InputField.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "./InputField.css";
+
+const InputField = ({ name, type, options, value, onChange }) => {
+  switch (type) {
+    case "text": {
+      return (
+        <input
+          id={name}
+          value={value}
+          onChange={onChange}
+          type={type}
+          placeholder="Type here..."
+          autoFocus
+          required
+        ></input>
+      );
+    }
+    case "radio": {
+      return options.map((o) => {
```

Comment:
What is o? option? Could it be named option?

Reply:
Great point, it makes sense to be more descriptive. Thanks!